Stellar-contract-cost-params.x

```diff
@@ -0,0 +1,60 @@
+%#include "xdr/Stellar-types.h"
```

Comment:
While I'm not a huge fan of jamming all the XDR together, this looks too granular compared to the current XDR organization. Probably it would make more sense to have all the configuration-related definitions be defined in the same file (like Stellar-network-configuration)

Reply:
Yeah I'm good with this suggestion. In particular the fees/limit, budget, cost params are all kind of related and make sense to live in the same file.

Reply:
I've split the config settings to a separate file.